resources/migrations/000_migrations.yaml

```diff
@@ -11501,6 +11501,52 @@ databaseChangeLog:
             columnName: icon
             defaultNullValue: "star"
 
+  - changeSet:
+      id: v43.00-061
+      author: qnkhuat
+      comment: >-
+        Added 0.43.0. Grant the 'All Users' Permissions Group readwrite perms for the Root Snippets Collection.
+      preConditions:
+        - onFail: MARK_RAN
+        # HACK: to prevent this migration from running on existing EE instances (#21940)
+        - sqlCheck:
+            expectedResult: 0
+            sql: >-
+              SELECT CASE
+                        /* Run for any new instsances */
+                        WHEN user_count = 0 THEN 0
+                        /* don't run for ee instances */
+                        WHEN token_count > 0 THEN 1
+                        /* Run for existing OSS instances */
+                        WHEN token_count = 0 then 0
+                    ELSE 1
+                    END AS RESULT
```

Comment:
This whole case can be simplified with

```sql
SELECT CASE
          WHEN (user_count > 0) AND (token_count) > 0 THEN 1
          ELSE 0
       END AS RESULT
```

But I prefer to have things explicit here